CRM/Report/Form/Event/Summary.php

```diff
@@ -262,8 +262,8 @@ public function buildColumnHeaders() {
       }
     }
 
-    $statusType1 = CRM_Event_PseudoConstant::participantStatus(NULL, 'is_counted = 1');
-    $statusType2 = CRM_Event_PseudoConstant::participantStatus(NULL, 'is_counted = 0');
+    $statusType1 = CRM_Event_PseudoConstant::participantStatus(NULL, 'is_counted = 1', 'label');
```

Comment:
This looks like a correct invocation of https://github.com/civicrm/civicrm-core/blob/master/CRM/Event/PseudoConstant.php#L137